test/tmp.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
@skeggse's preference is to utilize a proper internationalized domain with an active MX record. I have struck out on finding such a domain, so this approach was his backup plan. I'm guessing `tmp.js` was so that it doesn't become the permanent solution by default. :P